flash_notebooks/finetuning/image_classification.ipynb

```diff
@@ -0,0 +1,241 @@
+{
```

Comment:
<p>Flash Tasks have built-in DataModules that you can abuse to organize your data. Pass in a train, validation and test folders and Flash will take care of the rest.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done !</p>